.github/workflows/ci.yml

```diff
@@ -0,0 +1,97 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  mix_compile:
+    name: Compile (Erlang/OTP ${{matrix.otp}} | Elixir ${{matrix.elixir}})
+    runs-on: ubuntu-latest
+    container: hexpm/elixir:${{ matrix.elixir }}-erlang-${{ matrix.otp }}-alpine-3.11.6
```

Comment:
I recommend using the [erlef/setup-elixir](https://github.com/erlef/setup-elixir) action instead - it sets up rebar and hex for you and is the official action that sets up Elixir. Plus I think GH Actions are faster when not containerized, but that might be anecdata. Here's an example that worked for me: https://github.com/elixir-grpc/grpc/pull/189/files